CONTRIBUTING.md

```diff
@@ -1,261 +1,4 @@
-  - Visuals (diagrams, infographic design, icon design)
-- Management of communication tools (at the discretion of project maintainers)
-  - Mailing list moderation
-  - Discord management
-  - Calendar management
-- Event management
-  - Host community events
-  - Support summits and hackathons
-  - Staffing EDC booths at conferences
-
-## Project and Milestone Planning
-
-We use milestones to set a common focus for a period of 6 to 8 weeks. 
-The group of committers chooses issues based on customer needs and contributions we expect.
-
-### Milestones
-
-Milestones are organized at the [GitHub Milestones page](https://github.com/eclipse-edc/Connector/milestones).
-They are numbered in ascending order. There, contributors, users, and adopters can track the progress.
-
-Please note that the due date of a milestone does not imply any guarantee that all linked issued will 
-be resolved by then. 
-
-When closing the current milestone, issues that were not resolved within a milestone phase will be 
-reviewed to evaluate their relevance and priority, before being assigned to the next milestone.
-
-#### Issues
-
-Every issue that should be addressed during a milestone phase is assigned to it by using the 
-`Milestone` feature for linking both items. This way, the issues can easily be filtered by 
-milestones.
-
-#### Pull Requests
-
-Pull requests are not assigned to milestones as their linking to issues is sufficient to track 
-the relations and progresses.
-
-### Projects
-
-The [GitHub Projects page](https://github.com/eclipse-edc/Connector/projects)
-provides a general overview of the project's working items. Every new issue is automatically assigned
-to the ["Dataspace Connector" project](https://github.com/orgs/eclipse-edc/projects/3).
-It can be unassigned or moved to any other project that is provided.
-
-In every project, an issue passes four stages: `Backlog`, `In progress`, `Review in progress`, and `Done`,
-independent of their association to a specific milestone.
-
-### Releases
-
-Please find more information about our release approach [here](docs/developer/releases.md).
-
-## Contact Us
-
-If you have questions or suggestions, do not hesitate to contact the project developers via
-the [project's "dev" list](https://dev.eclipse.org/mailman/listinfo/edc-dev). 
-
-You may also want to join our [Discord server](https://discord.gg/n4sD9qtjMQ).
-
-There, we provide a biweekly meeting on fridays 2-3 p.m. (CET) to give any interested person the
-opportunity to get in touch with the committer team. We are meeting in the "general" voice channel.
-Find more details about the schedule [on GitHub](https://github.com/eclipse-edc/Connector/discussions/1303).
-
-_If you have a "contributor" or "committer" status, you will also have access to private channels._
+Thank you for your interest in contributing to this project. Please refer to the
+[contributing guidelines of the EDC project](https://github.com/eclipse-edc/docs/blob/main/CONTRIBUTING.md).
```

Comment:
should we also move the `pr_etiquette.md`?

Reply:
This has already been moved to the `docs` repo. Or what do you mean?